components/NewNFTBook.vue

```diff
@@ -533,18 +447,21 @@ const { wallet, signer } = storeToRefs(walletStore)
 const { newBookListing, updateEditionPrice } = bookStoreApiStore
 const { fetchStripeConnectStatusByWallet } = stripeStore
 const { getStripeConnectStatusByWallet } = storeToRefs(stripeStore)
+const { token } = storeToRefs(bookStoreApiStore)
+const nftStore = useNftStore()
+
+const UPLOAD_FILESIZE_MAX = 20 * 1024 * 1024
 
 const emit = defineEmits(['submit'])
 const router = useRouter()
 const route = useRoute()
-// params.editingClassId and params.editionIndex is only available when editing an existing class
-// query.class_id is only available when creating a new class
 const uploadStore = useUploadStore()
 const { getUploadFileData } = uploadStore
 const classId = ref(
-  route.params.editingClassId || (route.query.class_id as string)
+  route.params.classId || (route.query.class_id as string)
 )
-const editionIndex = ref(route.params.editionIndex as string)
+const editionIndex = ref(route.params.editionIndex as string) || ref(route.query.priceIndex as string)
+const newEditionIndex = ref(route.query.priceIndex)
```

Comment:
```suggestion
const newEditionIndex = ref(route.query.price_index)
```

Reply:
[🎨 Update query parameter in snake case](https://github.com/likecoin/nft-book-press/pull/332/commits/e29162b978afb388a14690857322bc6782e44c84)